src/coreComponents/physicsSolvers/SolverBase.cpp

```diff
@@ -290,32 +290,34 @@ real64 SolverBase::setNextDt( real64 const & currentDt,
   real64 const nextDtNewton = setNextDtBasedOnNewtonIter( currentDt );
   real64 const nextDtStateChange = setNextDtBasedOnStateChange( currentDt, domain );
 
-  if( nextDtNewton < nextDtStateChange ) // time step size decided based on convergence
+  if( nextDtNewton < nextDtStateChange )      // time step size decided based on convergence
   {
     integer const iterDecreaseLimit = m_nonlinearSolverParameters.timeStepDecreaseIterLimit();
     integer const iterIncreaseLimit = m_nonlinearSolverParameters.timeStepIncreaseIterLimit();
     if( nextDtNewton > currentDt )
     {
-      GEOS_LOG_LEVEL_RANK_0( 1, GEOS_FMT( "{}: Newton solver converged in less than {} iterations, time-step required will be increased.",
-                                          getName(), iterIncreaseLimit ) );
+      GEOS_LOG_LEVEL_RANK_0( 1, GEOS_FMT(
+                               "{}: Newton solver converged in less than {} iterations, time-step required will be increased.",
+                               getName(), iterIncreaseLimit ));
     }
     else if( nextDtNewton < currentDt )
     {
-      GEOS_LOG_LEVEL_RANK_0( 1, GEOS_FMT( "{}: Newton solver converged in more than {} iterations, time-step required will be decreased.",
-                                          getName(), iterDecreaseLimit ) );
+      GEOS_LOG_LEVEL_RANK_0( 1, GEOS_FMT(
+                               "{}: Newton solver converged in more than {} iterations, time-step required will be decreased.",
+                               getName(), iterDecreaseLimit ));
     }
   }
-  else // time step size decided based on state change
+  else         // time step size decided based on state change
   {
     if( nextDtStateChange > currentDt )
     {
       GEOS_LOG_LEVEL_RANK_0( 1, GEOS_FMT( "{}: Time-step required will be increased based on state change.",
-                                          getName() ) );
+                                          getName()));
```

Comment:
how is uncrustify not cleaning things like that? @tbeltzun would it after your PR?